include/wx/generic/gridsel.h

```diff
@@ -151,6 +162,16 @@ class WXDLLIMPEXP_CORE wxGridSelection
     wxGrid                              *m_grid;
     wxGrid::wxGridSelectionModes        m_selectionMode;
 
+    // Used by wxGrid::DrawSelectionOverlay() to draw a:
+    //
+    // - Simple rectangle (using wxDC::DrawRectangle() if it is empty and the bounding box is valid.
+    // - Simple polygon (using wxDC::DrawPolygon()) if it represents a simple polygon.
+    // - Poly-polygon (using wxDC::DrawPolyPolygon()) if it consists of multi-polygons.
```

Comment:
Really minor, but I'd say:
```suggestion
    // - Poly-polygon (using wxDC::DrawPolyPolygon()) if it consists of multiple polygons.
```